doc/data_format.rst

```diff
@@ -678,14 +678,18 @@ Offset     Length  Version  Description
   59+s      4       1-      Entrance fee.
   63+s      4       1-      Monthly cost of having the ride.
   67+s      4       1-      Additional monthly cost of having an opened ride.
-  71+s      4       1-      Text of the ride (reference to a TEXT block).
-  75+s                      Total length.
+  71+s      4       2-      Number of guests that can use the ride at the same time.
+  75+s      4       2-      Duration of the ride's idle phase in milliseconds.
+  79+s      4       2-      Total duration of the ride's working phase in milliseconds.
```

Comment:
I am guessing that idle duration and working duration are assumed to be fixed values for now, and they will change in the future.

However, you also have animation timing here, so there needs to be statement what happens if the animation specification and the duration here are in conflict.

Reply:
Added info about the requirements. There should be no need to change the specification here in the future though:
- The duration how long the ride idles can be heavily controlled by the player in RCT. You can set a minimum and maximum idle time, and specify that the ride shall start working on full load or half load etc. Any such setting overrides the duration from the RCD. But if the player says Any Load with no min/max time then the program should use the default value stated in the RCD.
- For the working phase, some rides allow the user to specify how many cycles the ride takes. Then the working duration from the RCD refers to 1 cycle, and the actual duration is the product of this base duration and the player's setting. I could already add space for the min and max numbers of cycles per working phase to the file definition, but as this feature is not of very high priority to me they would lie around unused for a while so I'd rather wait with adding them until several more features have been implemented. (The next steps in my plans are to allow building entrances and exits, and then to let guests actually enter rides.)